app/javascript/flavours/polyam/api/interactions.ts

```diff
@@ -30,7 +30,8 @@ export const apiGetQuotes = async (statusId: string, url?: string) => {
 
 // Polyam: Reactions
 
-export const apiReact = (statusId: string, name: string) =>
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const apiReact = (statusId: string, name: string, url?: string) =>
```

Comment:
I don't like this.
The url param is not needed here and its only purpose is to be passed to the reducer.
I don't know how else to achieve this though without creating a headache for myself.